CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @wyrzyk
```

Comment:
Why only me?

Reply:
You're the most knowledgeable in this module :)
If you feel there should be more code owners, we can always add them